vms/platformvm/vm_regression_test.go

```diff
@@ -1029,7 +1030,7 @@ func TestValidatorSetAtCacheOverwriteRegression(t *testing.T) {
 		extraNodeID,
 		ids.GenerateTestShortID(),
 		reward.PercentDenominator,
-		[]*secp256k1.PrivateKey{keys[0]},
+		[]*secp256k1.PrivateKey{ts.Keys[1]},
```

Comment:
changed since, with fixed fork time, Key[0] does not have enough balance to create the tx.
Got the following error
`couldn't generate tx inputs/outputs: insufficient funds (unlocked, locked) (0, 499990000) but need (0, 500000000)`